packages/stark-ui/src/modules/route-search/components/route-search.component.html

```diff
@@ -0,0 +1,58 @@
+<div *ngIf="direction === 'left'">
```

Comment:
As we discussed, instead of duplicating the HTML just to put the search button to the left or to the right, we can do this only with css: [flex-direction: row-reverse](https://css-tricks.com/almanac/properties/f/flex-direction/)